widget/embedded/src/components/Quote/Quote.types.ts

```diff
@@ -13,3 +14,20 @@ export type QuoteProps = {
   output: { value: string; usdValue?: string };
   expanded?: boolean;
 };
+
+export type QuoteTriggerProps = {
+  quoteRef: React.MutableRefObject<HTMLButtonElement | null>;
+  recommended: boolean;
+  setExpanded: React.Dispatch<React.SetStateAction<boolean | undefined>>;
+  steps: Step[];
+  numberOfSteps: number;
```

Comment:
I believe this property is unnecessary since you can obtain it by utilizing steps.length within the component.

Reply:
fixed